### DISCUSSION TITLE
Quiz Data Wrangling & Visualization

### DISCUSSION THREAD

**Discussion by ttiarasakti** [teamalgoritma/community#933] [Q&A] [answered]
Selamat Sore,
Saya mau tanya untuk nomor 3 dan 5, jawaban hasil analisis saya kok gaada di opsi jawabannya ya? Apa ada kesalahan di code saya?
Begitu juga jawaban pada no 5
![image](https://github.com/user-attachments/assets/fce15b90-5d55-42ea-9868-d9a111e5586c)

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear Ibu @ttiarasakti 
Terima kasih telah bertanya melalui platform Github Discussion Algoritma dan terima kasih atas kesediaannya untuk menunggu respon dari kami.

Terkait pertanyaan Ibu, pastikan ibu mengikuti kembali spesifikasi yang diminta oleh soal bu. Pastikan ibu sudah melakukan subsetting berdasarkan lokasi yang diminta dan ibu melakukan proses pembuatan dataframe `june_outliers` mengikuti step-step pada soal.

Sekian yang dapat kami sampaikan, bu
Semoga dapat membantu.

Best Regards,
Team Algoritma

**Reply by ttiarasakti** [reply]
Baik mba..
